File: buildSrc/src/main/java/org/briarproject/briar/desktop/builddata/FileBuilder.java
package org.briarproject.briar.desktop.builddata;

class FileBuilder {

	private static String nl = System.getProperty("line.separator");

	private StringBuilder buffer = new StringBuilder();

	void append(String string) {
		buffer.append(string);
	}

	void line() {
		buffer.append(nl);
	}

	void line(String string) {
		buffer.append(string);
		buffer.append(nl);
	}

	public String toString() {
		return buffer.toString();
	}

}

File: buildSrc/src/main/kotlin/org/briarproject/briar/desktop/builddata/FileBuilder.kt
package org.briarproject.briar.desktop.builddata

internal class FileBuilder {
    private val buffer = StringBuilder()

    fun line() {
        buffer.append(nl)
    }

    fun line(string: String?) {
        buffer.append(string)
        buffer.append(nl)
    }

    override fun toString(): String {
        return buffer.toString()
    }

    companion object {
        private val nl = System.getProperty("line.separator")
    }
}
